ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1082,18 +1095,22 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
                           accessoryType: TableViewCellAccessoryType = .none) {
         if let attributedTitle = attributedTitle {
             self.attributedTitle = attributedTitle
+            isAttributedTitleSet = true
         } else {
             self.attributedTitle = nil
             titleLabel.text = title
         }
         if let attributedSubtitle = attributedSubtitle {
             self.attributedSubtitle = attributedSubtitle
+            isAttributedSubtitleSet = true
         } else {
             self.attributedSubtitle = nil
             subtitleLabel.text = subtitle
         }
+
         if let attributedFooter = attributedFooter {
             self.attributedFooter = attributedFooter
+            isAttributedFooterSet = true
         } else {
             self.attributedFooter = nil
             footerLabel.text = footer
```

Comment:
does that mean in the else case we should set isAttributedFooterSet false? same question for title and subtitle

Reply:
The default is always set at false, and setup is called once in the beginning. Should I add it as insurance?

Reply:
yes! because nothing stops clients to call setup on the cell multiple times